coursera_autograder/commands/update_resource_limits.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+# Copyright 2015 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Coursera's asynchronous grader command line SDK.
+
+You may install it from source, or via pip.
+"""
+
+from coursera_autograder.commands import common
+from coursera_autograder.commands import oauth2
+import logging
+import requests
+import urllib.parse
+
+def command_update_resource_limits(args):
+    "Implements the update_resource_limits subcommand"
+
+    oauth2_instance = oauth2.build_oauth2(args)
+    auth = oauth2_instance.build_authorizer()
+
+    s = requests.Session()
+    s.auth = auth
+
+    course_branch_id = args.course.replace("~", "!~") if "authoringBranch" in args.course else args.course
+    course_branch_item = '%s~%s' % (course_branch_id, args.item)
+
+    params = 'id=%s&partId=%s' % (course_branch_item, args.part)
+    body = {
+        "reservedCpu": int(args.cpu) if args.cpu != None else None, 
+        "reservedMemory": int(args.memory) if args.memory != None else None, 
+        "wallClockTimeout": int(args.timeout) if args.timeout != None else None
+        }
+    result = s.post(args.updateGraderResourceLimits_endpoint, params = params, json = body)
+    if result.status_code == 404:
+        logging.error(
+            '\nUnable to find executor with part id %s in item %s in course %s.\n'
```

Comment:
How about this? 404, could mean either part doens't exist, or part doesn't have executor id?
```suggestion
            '\nUnable to find the part or executor with part %s in item %s in course %s.\n'
```

Reply:
Good point. I'll also update the wording from executor to grader to make it consistent with everything else.